### DISCUSSION TITLE
PostgreSQLSeverity.fatal 3D000: database "mypod" does not exist

### DISCUSSION THREAD

**Discussion by Thierrynjike** [serverpod/serverpod#116] [Q&A] [answered]
Hello, 

i am just starting with serverpod and when I run the mypod/mypod_server/bin/main.dart file, i get an error saying that the mypod database does not exist. Please could anyone tell me what should I do ?
![serverpod issue](https://user-images.githubusercontent.com/39903360/160760447-40348db4-88aa-4706-96e9-b8bd4a2cb248.png)

**Comment by vlidholt** [comment, marked as answer]
It seems like you haven't started Postgres and Redis which are required for running Serverpod. There are two ways you can do this. Your first option:

1. cd into `mypod/mypod_server`
2. Run `serverpod run`

Second way. Start the docker container manually then run serverpod like you tried to do:

1. cd into `mypod/mypod_server`
2. Start Docker: `docker-compose up -d --build`
3. Start Serverpod: `dart bin/main.dart`

More information here: https://docs.serverpod.dev/
